aten/src/ATen/native/Batching.cpp

```diff
@@ -70,27 +70,15 @@ static std::pair<Tensor,int64_t> remove_existing_batch_dim(
   return std::make_pair(std::move(result_tensor), newly_exposed_logical_dim);
 }
 
-// Poor man's version of np.moveaxis. Moves the dimension at `dst` to `src`
-// while preserving the order of other existing dimensions.
-// We should probably add np.moveaxis (it is more general) to PyTorch. (#36048)
-// When we do, replace the following with it.
-static Tensor movedim(const Tensor& self, int64_t src, int64_t dst) {
+// at::movedim but may return the original tensor if dst is the same as src.
```

Comment:
It turns out the behaviour here deviates slightly from `at::movedim` but I'd say it's at least worth highlighting that.